iron-overlay-manager.html

```diff
@@ -385,6 +386,7 @@
      * @param {!Element} overlay1
      * @param {!Element} overlay2
      * @return {boolean}
+     * @suppress {missingProperties}
```

Comment:
if it was working, why would I need to send you a pull request? :wink: 

with the cast, chrome's code still fails to compile like this:
```
## /chromium/src/third_party/polymer/v1_0/components-chromium/iron-overlay-behavior/iron-overlay-manager-extracted.js:377: ERROR - Property alwaysOnTop never defined on o1
##       return !o1.alwaysOnTop && o2.alwaysOnTop;
##                  ^
```
with the suppression: compilation succeeds.

obviously somebody added this to fix their error, but Chrome likely has slightly different flag combo.

tl;dr - i can remove the cast if you'd like, but I don't understand who/how it's helping so didn't want to change it.